docs/USAGE.md

```diff
@@ -0,0 +1,282 @@
+# Usage
+
+**WARNING**: Please notice that no official release has been created for this repo yet, so installation instructions below would require you to build the code manually beforehand.
+
+## Configure the OpenTelemetry Tracing Library for .NET
+
+### Configuration values
+
+Use these environment variables to configure the tracing library:
+
+| Environment variable | Default value | Description |
```

Comment:
Can we change the table formatting to the one used here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md to be consistent?